package.json

```diff
@@ -23,44 +23,40 @@
     "build": "pkgu build",
     "clean": "rm -rf dist",
     "lint": "eslint . --ext .js,.ts",
-    "precommit:logger-preview": "ts-node --compiler-options '{\"module\":\"commonjs\"}' scripts/generateSvg.logger.teams.ts",
+    "precommit:logger-preview": "ts-node scripts/generateSvg.logger.teams.ts",
     "prepare": "husky install",
     "prepush:test": "jest --runInBand --onlyChanged --coverage=false",
     "test": "jest",
     "test:watch": "yarn test --watch",
     "tsc": "tsc --noEmit --project tsconfig.dev.json",
     "validate": "npm-run-all tsc lint test build"
   },
-  "peerDependencies": {
-    "web-vitals": "^2"
-  },
   "devDependencies": {
-    "@guardian/eslint-config": "0.6.0",
-    "@guardian/eslint-config-typescript": "0.6.0",
+    "@guardian/eslint-config": "1.0.0",
+    "@guardian/eslint-config-typescript": "1.0.1",
     "@guardian/pkgu": "0.6.0",
-    "@guardian/prettier": "0.7.0",
-    "@rollup/plugin-typescript": "8.2.5",
+    "@guardian/prettier": "1.0.0",
     "@semantic-release/github": "7.2.3",
-    "@types/jest": "27.0.1",
+    "@types/jest": "27.4.1",
     "@types/wcag-contrast": "3.0.0",
-    "conventional-changelog-conventionalcommits": "4.6.0",
-    "eslint": "7.32.0",
-    "husky": "7.0.2",
-    "jest": "27.0.6",
+    "conventional-changelog-conventionalcommits": "4.6.3",
+    "eslint": "8.9.0",
+    "husky": "7.0.4",
+    "jest": "27.5.1",
     "jest-fetch-mock": "3.0.3",
     "mockdate": "3.0.5",
     "npm-run-all": "4.1.5",
-    "prettier": "2.4.1",
-    "rollup": "2.56.3",
-    "rollup-plugin-size": "0.2.2",
-    "rollup-plugin-terser": "7.0.2",
+    "prettier": "2.5.1",
     "semantic-release": "17.4.5",
-    "ts-jest": "27.0.5",
-    "ts-node": "10.2.1",
+    "ts-jest": "27.1.3",
+    "ts-node": "10.5.0",
     "tslib": "2.3.1",
-    "typescript": "4.3.5",
+    "typescript": "4.0.2",
```

Comment:
Is this an intentional downgrade of `typescript`?

Reply:
yeah same principle - make sure we're not using TS features that consumers may not have available. since it's also JS-compatible we can't define TS as a peerDep, but at least we can say it works with v4